src/platforms/java/guides/jul/configuration/configuration-example.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Configuration Example
+sidebar_order: 20
+description: "Review an example of configuration for JUL."
+---
+
+<!-- TODO: link to correct site with explanation of each configuration property -->
```

Comment:
should we address this in this PR or just remove it?

Reply:
This has been added by @PeloWriter in Logback section, since JUL docs are very similar I think we should keep it here for the reference.